Backend/index.js

```diff
@@ -169,6 +169,34 @@ app.get("/listings/category/:category", async (req, res) => {
   }
 });
 
+app.get("/listings/subcategory/:subcategory", async (req, res) => {
```

Comment:
nit: i would have this subcategory code be in a separate commit from the enums feedback code in the future to make the commits even smaller in size and easier to review!